modules/persistent-storage-vsphere-dynamic-provisioning.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-vsphere.adoc
+
+[id="vsphere-dynamic-provisioning_{context}"]
+= Dynamically provisioning VMware vSphere volumes
+
+Dynamically provisioning VMware vSphere volumes is the preferred provisioning method. {product-title} installs a default StorageClass, named `thin`, that uses the `thin` disk format for provisioning volumes.
+
+.Prerequisites
+
+* Storage must exist in the underlying infrastructure before it can be mounted as a volume in {product-title}.
+
+.Procedure (UI)
+
+. In the {product-title} console, click *Storage* → *Persistent Volume Claims*.
+
+. In the persistent volume claims overview, click *Create Persistent Volume Claim*.
+
+. Define the required options on the resulting page.
+
+.. Select the `thin` StorageClass.
+
+.. Enter a unique name for the storage claim.
+
+.. Select the access mode to determine the read and write access for the created storage claim.
```

Comment:
Are there any requirements or limitations for the access modes?

Reply:
Access modes are discussed under the Persistent Storage general chapter, and are listed as a prerequisite to this assembly. I'm leaving this as is.